CHANGELOG.md

```diff
@@ -1,7 +1,12 @@
 # Changelog
 
-## Ongoing
+## v0.34.0
```

Comment:
Suggest bumping to .40 as per new features

Reply:
Why v0.40.0? To v0.34.0 is also a minor bump?